src/components/Wizard/WizardProgressBar/style.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components'
+import { Colors } from '../../../tokens'
+
+const WizardProgressBarContainerStyled = styled.div`
+  position: relative;
+  height: 2px;
+  width: calc(100% - 44px);
+  margin-bottom: 16.5px;
+  background-color: ${Colors.light.neutral.neutral200};
```

Comment:
Shouldn't this `Colors.light` pattern come after changing the settings?

Reply:
Light or dark? As the theme configuration adjustment will be implemented soon, I left it just light.